.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
```

Comment:
- [ Optional ] Please make sure that your mobile version home page matches the design 
 
| Your code  | Behance|
| ------------- | ------------- |
|  ![image](https://user-images.githubusercontent.com/74744787/212499933-a6a80917-4bdb-4349-bbdd-fda63bde09a4.png) | ![image](https://user-images.githubusercontent.com/74744787/212499940-b35946a3-94dd-4ee5-92a0-36390bd6a8b2.png)  |